app/controllers/admin/claims_controller.rb

```diff
@@ -9,4 +9,14 @@ def show
     @claim = Claim.find(params[:id])
     @check = @claim.check || Check.new
   end
+
+  def search
+    return unless params[:reference].present?
+
+    if (claim = Claim.find_by(reference: params[:reference]))
```

Comment:
Could we separate assignment from the conditional? I find that having assignments in conditionals can be confusing when scan reading the code.